kit/ChildSession.cpp

```diff
@@ -3209,6 +3209,11 @@ void ChildSession::loKitCallback(const int type, const std::string& payload)
         sendTextFrame("corelog: " + payload);
         break;
     }
+    case LOK_CALLBACK_TOOLTIP:
+    {
+        sendTextFrame("tooltip: " + payload);
```

Comment:
Might need to add `"tooltip:"` in `HTTPWSTest::testInactiveClient()`, in case it is generated by a test document.

Reply:
Thank you!
Let me merge this, and then I will try to create a follow-up implementing your suggestion.